src/Hl7.Fhir.Base/Serialization/BaseFhirSerializer.cs

```diff
@@ -39,7 +41,7 @@ protected ITypedElement MakeElementStack(Base instance, SummaryType summary, str
 
         protected ITypedElement MakeElementStack(Base instance, SummaryType summary, string[]? elements, bool includeMandatoryInElementsSummary)
         {
-            if (summary == SummaryType.False && elements == null) return instance.ToTypedElement(_modelInspector);
+            if (summary == SummaryType.False && elements == null) return instance.ToTypedElementLegacy(_modelInspector);
```

Comment:
Now I look at it, I wonder why this fails now we do have a model inspector. It should work with Definition present.